docs/testing/releases/590.md

```diff
@@ -0,0 +1,35 @@
+## Testing notes and ZIP for release 5.9.0
+
+Zip file for testing: [woocommerce-gutenberg-products-block.zip](https://github.com/woocommerce/woocommerce-gutenberg-products-block/files/7161568/woocommerce-gutenberg-products-block.zip)
+
+## Feature plugin and package inclusion in WooCommerce
```

Comment:
The changes listed here are not included in WC core, right?
```suggestion
## Feature plugin
```

Reply:
Yes correct I've removed the package inclusion wording here for 4668, but created a new heading for WC Core inclusion and moved the bug fix for product search under it - that'll be in WC Core.